app/views/partials/fileCheckGenericErrorMessage.scala.html

```diff
@@ -1,6 +1,7 @@
+@(consignmentRef: String)
 <p class="govuk-body">
-    One or more files you uploaded have failed our checks. Please
-    <a class="govuk-link" href="mailto:tdr@@nationalachives.gov.uk?subject=Problem with Results of checks">Contact us</a>
+    One or more files you uploaded have failed our checks. Contact us at
+    <a class="govuk-link" href="mailto:tdr@@nationalachives.gov.uk?subject=Ref: @consignmentRef Problem with Results of checks">tdr@@nationalachives.gov.uk</a>
```

Comment:
I was thinking that maybe there should be a hyphen in-between the "consignmentRef" and the "Problem with Results of checks" like `@consignmentRef  - Problem with Results of checks` just for clarity.

Reply:
Yeah that would make it easier to read :+1: